packages/block-editor/src/components/index.js

```diff
@@ -23,6 +23,7 @@ export { default as BlockColorsStyleSelector } from './color-style-selector';
 export { default as BlockEdit, useBlockEditContext } from './block-edit';
 export { default as BlockIcon } from './block-icon';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
+export { default as __experimentalBlockNavigation } from './block-navigation';
```

Comment:
It'd be great to call this `ListView` or similar. Maybe it could all be moved to a list-view folder leaving only the dropdown in the block navigation folder (which might also be a candidate for deprecation).

That is quite a big change, so definitely optional!

Reply:
This is actually a great idea. 🤔 

I'll try pushing a commit and see if git manages to keep the diff readable, otherwise I'll move to a follow up.

Reply:
Ooof, it went from +10-10 to +400-400 🤣